lib/sdam/topology_description.js

```diff
@@ -277,8 +277,22 @@ class TopologyDescription {
 }
 
 function topologyTypeForServerType(serverType) {
-  if (serverType === ServerType.Mongos) return TopologyType.Sharded;
-  if (serverType === ServerType.RSPrimary) return TopologyType.ReplicaSetWithPrimary;
+  if (serverType === ServerType.Standalone) {
+    return TopologyType.Single;
+  }
+
+  if (serverType === ServerType.Mongos) {
+    return TopologyType.Sharded;
+  }
+
+  if (serverType === ServerType.RSPrimary) {
+    return TopologyType.ReplicaSetWithPrimary;
+  }
+
+  if (serverType === ServerType.RSGhost || serverType === ServerType.Unknown) {
+    return TopologyType.Unknown;
+  }
+
```

Comment:
I personally think switches are cleaner for this sort of thing.

```
switch (serverType) {
  case ServerType.Standalone:
    return TopologyType.single;
  case ServerType.Mongos:
    return TopologyType.Sharded;
  case ServerType.RSPrimary:
    return TopologyType.ReplicaSetWithPrimary;
  default:
    return TopologyType.Unknown;
}
```

Thoughts?